compiler/src/dotty/tools/dotc/printing/RefinedPrinter.scala

```diff
@@ -1060,7 +1060,7 @@ class RefinedPrinter(_ctx: Context) extends PlainPrinter(_ctx) {
     if (sym.isImport)
       sym.infoOrCompleter match {
         case info: Namer#Completer => return info.original.show
-        case info: ImportType => return s"import $info.expr.show"
+        case info: ImportType => return s"import ${info.expr.show}"
```

Comment:
I sense a useful lint for a third-party linter!

Reply:
🤯 wow, seeing this in dotc... I had to check: this dates back to 2016

Out of curiosity I started my custom "third-party linter"
```
git grep 's".*\$[a-zA-Z0-9_-]*\..*"'|grep -v '{'
```
and didn't find any other obvious cases like this one. 

Going through those does demonstrates however that it'd be quite tricky to judge automatically if it's a bug or not... e.g. consider these:
```
community-build/src/scala/dotty/communitybuild/projects.scala:  override val testCommand = s"$baseCommand.test"
compiler/src/dotty/tools/dotc/printing/PlainPrinter.scala:    s"import $exprStr.$selectorStr"
compiler/src/dotty/tools/dotc/util/Spans.scala:    //assert(start <= end || start == 1 && end == 0, s"$start..$end")
compiler/src/dotty/tools/runner/ScalaClassLoader.scala:      throw new NoSuchMethodException(s"$objectName.main is not static")
```